stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -971,10 +909,6 @@ public class MessageListView : ConstraintLayout {
     public fun interface ConfirmDeleteMessageHandler {
         public fun onConfirmDeleteMessage(
             message: Message,
-            title: String,
-            description: String,
-            positiveText: String,
-            negativeText: String,
```

Comment:
Removed these as whoever customizes this handler can have arbitrary UI, and these parameters aren't generally useful for them.